src/components/Main.js

```diff
@@ -30,9 +30,7 @@ export default class Main extends React.Component {
     // Create an observer to notify the changes in the scene
     var target = document.querySelector('a-scene');
     var observer = new MutationObserver(function (mutations) {
```

Comment:
This is me again trying to understand how this works. Are we using MutationObservers to check for components updates. We're not using the event `componentchanged` emitted by entities?